tests/config-tests.js

```diff
@@ -147,3 +150,28 @@ describe('Config schema tests', () => {
         });
     }
 });
+
+describe('Config schema tests', () => {
```

Comment:
You've got the test in the `describe` block, instead of the `it` block. Does it still work?

I think we can just add an `it` with a description of what we're testing for, e.g.
```javascript
describe('Platform overrides', () => {
  it('each overriden feature should include all sub features from base feature', () => {
```